near/src/main.rs

```diff
@@ -18,7 +18,7 @@ use near::{get_default_home, get_store_path, init_configs, load_config, start_wi
 use near_primitives::types::Version;
 
 fn init_logging(verbose: Option<&str>) {
-    let mut env_filter = EnvFilter::new("tokio_reactor=info,near=info,stats=info");
+    let mut env_filter = EnvFilter::new("tokio_reactor=info,near=info,stats=info,telemetry=info");
```

Comment:
why do we need to include telemetry logs?

Reply:
To make them visible... We have broken Telemetry 3 times since we introduced it, and every single time it took us quite some time to notice that.

Reply:
I am not sure whether we want to do this by default. It seems useless for most people.

Reply:
It will only print any log if there is an issue with telemetry, so most of the time it is going to be silent...

Reply:
Okay then